cmake/projects/libsodium/hunter.cmake

```diff
@@ -17,10 +18,14 @@ hunter_add_version(
     00a999629f1f8f500cef29818135f8f889f44e41
 )
 
+hunter_configuration_types(libsodium CONFIGURATION_TYPES Release)
 if(MSVC)
     hunter_pick_scheme(DEFAULT url_sha1_libsodium_msbuild)
 else()
     hunter_pick_scheme(DEFAULT url_sha1_libsodium_autogen_autotools)
```

Comment:
Let me investigate this, somewhere along this change I did a `git clean`, so I ended up losing some things that could have caused this.